morphir/runtime/src/org/finos/morphir/runtime/Distributions.scala

```diff
@@ -20,22 +20,6 @@ import org.finos.morphir.ir.distribution.Distribution
 import org.finos.morphir.ir.distribution.Distribution.Library
 import zio.Chunk
 
-//TODO: Error hierarchy and code should reflect possibility of specification lookup
-sealed abstract class LookupError(msg: String) extends Exception(msg) {
```

Comment:
Moved directly into `morphir/org.finos.morphir` since it's used in MorphirRuntimeError